package.json

```diff
@@ -25,7 +25,7 @@
     "core-object": "^1.1.0",
     "ember-cli": "0.2.7",
     "ember-cli-app-version": "0.3.3",
-    "ember-cli-aptible-shared": "0.0.24",
+    "ember-cli-aptible-shared": "0.0.25",
```

Comment:
This is causing the errors that Travis is having, see https://github.com/aptible/dashboard.aptible.com/pull/370 for confirmation.  The issue is that we are using `Ember.run.later` to poll for updates in provisioning, and that causes extra requests to the server.